libpod/define/container.go

```diff
@@ -38,3 +38,11 @@ const (
 	// ContainerInitPath is the default path of the mounted container init.
 	ContainerInitPath = "/run/podman-init"
 )
+
+// Kubernetes Kinds
+const (
+	// A Pod kube yaml spec
+	PodKind = "pod"
```

Comment:
Let's prefix these consts  as done for other objects.: `K8sKindPod` and `K8sKindDeployment`

This namespacing will prevent name conflicts in the future.

Reply:
Fixed